File: app/src/main/java/com/do_big/diginotes/utils/Converters.java
package com.do_big.diginotes.utils;

import androidx.room.TypeConverter;
import androidx.room.TypeConverters;

import java.util.Date;

public class Converters {

    @TypeConverter
    public static Date fromTimestamp(Long value){
        return  value==null?null:new Date(value);

    }
    @TypeConverter
    public static Long datetoTimestamp(Date date){
        return date==null?null:date.getTime();

    }

}

File: app/src/main/java/com/do_big/diginotes/utils/Converters.kt
package com.do_big.diginotes.utils

import androidx.room.TypeConverter
import java.util.Date

object Converters {
    @JvmStatic
    @TypeConverter
    fun fromTimestamp(value: Long?): Date? {
        return if (value == null) null else Date(value)
    }

    @JvmStatic
    @TypeConverter
    fun datetoTimestamp(date: Date?): Long? {
        return date?.time
    }
}
